docs/content/examples/map-editor.tsx

```diff
@@ -1,5 +1,14 @@
+import { encodeEffects } from '@deities/apollo/Effects.tsx';
 import { Sniper } from '@deities/athena/info/Unit.tsx';
 import MapEditor from '@deities/hera/editor/MapEditor.tsx';
+import {
+  MapCreateVariables,
+  MapObject,
+  MapUpdateVariables,
+} from '@deities/hera/editor/Types.tsx';
+import useLocation from '@deities/ui/hooks/useLocation.tsx';
+import { useEffect, useState } from 'react';
+import toSlug from '../../../hephaestus/toSlug.tsx';
```

Comment:
Can you add `"@deities/hephaestus": "workspace:*",` to the `docs/package.json` file, run `pnpm install` and then import via `@deities/hephaestus` instead?

Sorry it wasn't a dep before and ideally there are no relative imports outside of a package.